packages/core-backend/src/BackendWebSocketService.ts

```diff
@@ -1300,14 +1349,48 @@ export class BackendWebSocketService {
   // =============================================================================
 
   /**
-   * Schedules a reconnection attempt with exponential backoff
+   * Schedules a connection attempt with exponential backoff and jitter
+   *
+   * This method is used for automatic reconnection with backoff:
+   * - Prevents duplicate reconnection timers (idempotent)
+   * - Applies exponential backoff based on previous failures
+   * - Adds jitter (±25%) to prevent thundering herd problem
+   * - Used ONLY for automatic retries, not user-initiated actions
+   *
+   * Call this from:
+   * - connect() catch block (on connection failure)
+   * - #handleClose() (on unexpected disconnect)
+   *
+   * For user-initiated actions (sign in, unlock), call connect() directly instead.
+   *
+   * If a reconnect is already scheduled, this is a no-op to prevent:
+   * - Orphaned timers (memory leak)
+   * - Inflated reconnect attempts counter
+   * - Prematurely long delays
    */
-  #scheduleReconnect(): void {
+  #scheduleConnect(): void {
```

Comment:
Nit: It seems this is always called after connecting once or attempting to connect once, maybe `#scheduleReconnect` is a better name?